apps/etterlatte-behandling/src/main/kotlin/Contekst.kt

```diff
@@ -73,32 +66,27 @@ class SaksbehandlerMedEnheterOgRoller(
         return identifiedBy.hentTokenClaims(AZURE_ISSUER)!!.getStringClaim("NAVident")
     }
 
-    fun erSuperbruker() = name() in (saksbehandlereMedTilgangTilAlleEnheter)
-
-    fun enheter() =
-        if (saksbehandlerMedRoller.harRolleNasjonalTilgang()) {
-            Enheter.nasjonalTilgangEnheter()
-        } else {
-            try {
-                runBlocking {
-                    navAnsattKlient.hentEnhetForSaksbehandler(name()).map { it.id }
-                }
-            } catch (e: Exception) {
-                throw HentEnhetException("Henting av enheter feilet. Sjekk on-prem status(fss)", e)
+    fun enheterMedSkrivetilgang() =
+        try {
+            runBlocking {
+                navAnsattKlient.hentEnhetForSaksbehandler(name())
+                    .map { it.id }
+                    .filter { Enheter.enheterMedSkrivetilgang().contains(it) }
             }
+        } catch (e: Exception) {
+            throw HentEnhetException("Henting av enheter feilet. Sjekk on-prem status(fss)", e)
         }
 
-    override fun harSkrivetilgang(): Boolean {
-        val enheter = enheter()
-        val skrivetilgangEnhetsnummere = Enheter.enheterMedSkrivetilgang()
-        return enheter.any { skrivetilgangEnhetsnummere.contains(it) }
-    }
+    // TODO - EY-3441 - lesetilgang for forvaltningsutviklere
```

Comment:
Er det oss?

Reply:
Ja - det er en minne til meg selv for øyeblikket. Vi kan ikke bare sett i gang med det frem til det er gått opp ordentlig. Kommentar fjernes når PR er klar